test/parallel/test-http-flush-headers.js

```diff
@@ -5,7 +5,7 @@ const http = require('http');
 
 const server = http.createServer();
 server.on('request', function(req, res){
-  assert(req.headers['foo'], 'bar');
+  assert(req.headers['foo'] === 'bar');
```

Comment:
Ah I see now. Maybe it would be better to change this to `assert.equal()` instead, that way if the assertion should fail, it will show the two non-matching values in the message instead of just "false."